packages/neuron-ui/src/utils/formatters.ts

```diff
@@ -81,8 +81,13 @@ export const CKBToShannonFormatter = (amount: string, uint: CapacityUnit) => {
   }
 }
 
-export const shannonToCKBFormatter = (shannon: string) => {
-  const sign = shannon.startsWith('-') ? '-' : ''
+export const shannonToCKBFormatter = (shannon: string, showSign?: boolean) => {
```

Comment:
Is `showPositiveSign` more precise?

Reply:
Updated in https://github.com/nervosnetwork/neuron/pull/884/commits/e40a276ed25b333a3098ad1b02467c9a1455cdd1